data/data/manifests/bootkube/cvo-overrides.yaml.template

```diff
@@ -8,6 +8,6 @@ spec:
   upstream: https://amd64.origin.releases.ci.openshift.org/graph
   channel: stable-4
 {{- else }}
-  channel: stable-4.10
+  channel: stable-4.11
```

Comment:
Can a test be added so that every time the test runs in a new branch it will fail unless the version is added in this file? It is bad situation is someone has to remember every time to send a PR manually

Reply:
I meant unit test.